src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,200&family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
better to not reset these values in `*`

Reply:
Not fixed. Don't reset margins and padding using * selector. Specify the exact tags you want to reset
![image](https://user-images.githubusercontent.com/96409563/224535217-042ff711-29ba-4f1b-b2aa-e77acda733f5.png)
```
body,
nav, 
list {
  margin: 0;
  padding: 0;
}
```